### PR TITLE
Fix #87: pick up IP in US for heartbeat

### LINKED ISSUE #87: Heartbeat is returning 503 even though geoip lookup is working
Shelling to a container shows the geoip db is present. Checking the location metric shows tags for many different countries. However, when you hit https://classify-client.services.mozilla.com/__heartbeat__ (or dev or stage version) it returns a 503 response with the body `{"geoip":false}`.

I think this is because the [heartbeat code](https://github.com/mozilla/classify-client/blob/8e379de554ef7fd964ab3f2a4de1ffe7dd301a7c/src/endpoints/dockerflow.rs#L20-L32) tries to geocode 1.2.34 and verify this is a US address. When I plug that address into https://www.maxmind.com/en/geoip-demo, it is reported as being in Russia.

### PR DESCRIPTION
**Pull request by leplatrem**
...I don't know if that's the most sustainable way of fixing #87

### CODE REVIEW

**Comment by leplatrem** [comment]
The fix for the lint issue lives in #85

**Comment by mythmon** [comment]
Over in mozilla/normandy#2044 I made the check assert that *any* country was returned. In the Python version IPs that aren't in the DB either throw an error or return `None`. What do you think about using a similar idea here? I don't feel strongly about either way.

File: src/endpoints/dockerflow.rs [hunk 1]
```diff
@@ -26,7 +26,7 @@ pub fn heartbeat(app_data: Data<EndpointState>) -> Result<HttpResponse, Classify
             Some(country_info) => country_info
                 .country
                 .and_then(|country| country.iso_code)
-                .and_then(|iso_code| Some(Ok(iso_code == "US")))
+                .and_then(|iso_code| Some(Ok(iso_code.len() > 0)))
```
Comment (mythmon) [inline src/endpoints/dockerflow.rs]: Clippy wants this to be

```suggestion
                .and_then(|iso_code| Some(Ok(!iso_code.is_empty())))
```

**Comment by mythmon** [comment]
This should work now

bors r+

**Pull request merged**